dailymotion-player-objc/DMPlayerViewController.m

```diff
@@ -282,6 +298,23 @@ - (void)api:(NSString *)method {
     [self api:method arg:nil];
 }
 
+- (NSString *) getAPIReadyWarnMessageForMethod:(NSString *)method {
```

Comment:
ObjC style is to not use `get` prefixes on methods, just `APIReadyWarnMessageForMethod:`

Note to others (out of scope for this PR): Bring the API to the modern ages by adding nullability annotations which gives valuable compiler warnings and makes Swift interop nicer